scrapers/TimTales.yml

```diff
@@ -0,0 +1,65 @@
+name: "TimTales"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.timtales.com/videos/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.timtales.com/the-men/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title: $info/h1/text()
+      Date:
+        selector: $info/p[@class="date"]/text()
+        replace:
+          - regex: \s–\s.*$
+            with:
+        parseDate: January 02, 2006
+      Details:
+        selector: $info/p[@class="bodytext"]/text()
+        concat: " "
+      Tags:
+        Name: $info/p[@class="categories" and contains(text(), "Categories:")]/a/text()
+      Performers:
+        Name: $info/p[@class="categories" and contains(text(), "Men:")]/a/text()
```

Comment:
for some scenes , performers aren't anchors
eg `https://www.timtales.com/videos/2-for-1/` `https://www.timtales.com/videos/rocco-fucks-alejandro-dumas/`
The below seems to fix that
```yaml
      Performers:
        Name: 
          selector: $info/p[@class="categories" and contains(text(), "Categories:")]
          postProcess:
            - replace:
               - regex: "Men:"
                 with: ""
               - regex: "(,)\\s+"
                 with: "$1"
          split: ","
```